apps/condo/domains/ticket/access/Ticket.js

```diff
@@ -14,22 +14,30 @@ const { getById, find } = require('@core/keystone/schema')
 const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
 const { RESIDENT, STAFF } = require('@condo/domains/user/constants/common')
 
-async function canReadTickets ({ authentication: { item: user } }) {
+async function canReadTickets ({ authentication: { item: user }, context }) {
     if (!user) return throwAuthenticationError()
     if (user.deletedAt) return false
 
     if (user.isSupport || user.isAdmin) return {}
 
     if (user.type === RESIDENT) {
         const residents = await find('Resident', { user: { id: user.id }, deletedAt: null })
+
+        if (isEmpty(residents)) return false
+
         const organizationsIds = compact(residents.map(resident => get(resident, 'organization')))
+        const residentAddressOrStatement = residents.map(resident =>
+            ({ AND: [{ canReadByResident: true, contact: { phone: user.phone } }, { property: { id: resident.property } }, { unitName: resident.unitName }] }))
```

Comment:
why you don't use just `{ canReadByResident: true, contact: { phone: user.phone }, property: { id: resident.property }, unitName: resident.unitName }` ?

Reply:
Because these conditions are put in the `OR` statement, which works like a `sql join`.
If do not combine in AND, then it will work out how many OR expressions